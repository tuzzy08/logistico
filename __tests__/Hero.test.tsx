// eslint-disable-next-line no-use-before-define
import * as React from 'react'
import { render } from '@testing-library/react'
import Hero from '../components/Hero'

describe('Testing Hero component', () => {
  it('Should render correctly', () => {
    const { container } = render(<Hero />)
    expect(container).toMatchSnapshot()
  })
})
