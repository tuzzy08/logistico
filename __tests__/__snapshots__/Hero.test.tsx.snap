// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Hero component Should render correctly 1`] = `
.emotion-0 {
  max-width: 7xl;
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-1>*:not(style)~*:not(style) base {
  -webkit-margin-start: 0;
  margin-inline-start: 0;
}

.emotion-1>*:not(style)~*:not(style) base marginTop {
  base: 8px;
  md: 10px;
}

.emotion-1>*:not(style)~*:not(style) md {
  margin-top: 0;
}

.emotion-1>*:not(style)~*:not(style) md marginStart {
  base: 8px;
  md: 10px;
}

.emotion-1 flexDirection {
  base: column;
  md: row;
}

.emotion-1 py {
  base: 20px;
  md: 28px;
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.emotion-2>*:not(style)~*:not(style) {
  -webkit-margin-start: 0;
  margin-inline-start: 0;
}

.emotion-2>*:not(style)~*:not(style) marginTop {
  base: 5px;
  md: 10px;
}

.emotion-3 {
  line-height: 1.1;
  font-weight: 600;
}

.emotion-3 fontSize {
  base: 3xl;
  sm: 4xl;
  lg: 6xl;
}

.emotion-4 {
  position: relative;
}

.emotion-4::after {
  content: '';
  width: full;
  height: 30%;
  position: absolute;
  bottom: 1px;
  left: 0;
  background: green.400;
  z-index: -1;
}

.emotion-5 {
  line-height: 1.1;
  font-weight: 300;
}

.emotion-5 fontSize {
  base: 3xl;
  sm: 4xl;
  lg: 6xl;
}

.emotion-6 {
  color: green.300;
}

.emotion-7 {
  box-shadow: md;
  border-width: 1px;
  border-radius: md;
  padding: 5px;
}

.emotion-8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-8>*:not(style)~*:not(style) {
  margin-top: 0.5rem;
  -webkit-margin-start: 0;
  margin-inline-start: 0;
}

.emotion-10 {
  display: block;
  text-align: start;
}

.emotion-11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.emotion-11>*:not(style)~*:not(style) {
  -webkit-margin-start: 2px;
  margin-inline-start: 2px;
  margin-top: 0;
}

.emotion-12 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.emotion-12>*:not(style)~*:not(style) {
  -webkit-margin-start: 0.5rem;
  margin-inline-start: 0.5rem;
  margin-top: 0;
}

.emotion-14 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-appearance: none;
  -moz-appearance: none;
  -ms-appearance: none;
  appearance: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-transition: all 250ms;
  transition: all 250ms;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  position: relative;
  white-space: nowrap;
  vertical-align: middle;
  outline: none;
  width: auto;
}

.emotion-15 {
  width: 1em;
  height: 1em;
  display: inline-block;
  line-height: 1em;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  color: currentColor;
  vertical-align: middle;
}

.emotion-16 {
  padding-bottom: 4px;
  margin-bottom: 4px;
}

.emotion-17 {
  list-style-type: none;
  background: white;
  border-radius: 4px;
  box-shadow: 6px 5px 8px rgba(0,50,30,0.02);
}

.emotion-18 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-18>*:not(style)~*:not(style) base {
  -webkit-margin-start: 0;
  margin-inline-start: 0;
}

.emotion-18>*:not(style)~*:not(style) base marginTop {
  base: 4px;
  sm: 6px;
}

.emotion-18>*:not(style)~*:not(style) sm {
  margin-top: 0;
}

.emotion-18>*:not(style)~*:not(style) sm marginStart {
  base: 4px;
  sm: 6px;
}

.emotion-18 flexDirection {
  base: column;
  sm: row;
}

.emotion-19 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-appearance: none;
  -moz-appearance: none;
  -ms-appearance: none;
  appearance: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-transition: all 250ms;
  transition: all 250ms;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  position: relative;
  white-space: nowrap;
  vertical-align: middle;
  outline: none;
  width: auto;
  border-radius: full;
  font-weight: normal;
  padding-left: 6px;
  padding-right: 6px;
  color: white;
  background: green.300;
}

.emotion-19:hover,
.emotion-19[data-hover] {
  border-color: green;
}

.emotion-20 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-appearance: none;
  -moz-appearance: none;
  -ms-appearance: none;
  appearance: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-transition: all 250ms;
  transition: all 250ms;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  position: relative;
  white-space: nowrap;
  vertical-align: middle;
  outline: none;
  width: auto;
  border-radius: full;
  font-weight: normal;
  padding-left: 6px;
  padding-right: 6px;
}

.emotion-21 {
  -webkit-margin-end: 0.5rem;
  margin-inline-end: 0.5rem;
}

.emotion-22 {
  width: 4px;
  height: 4px;
  display: inline-block;
  line-height: 1em;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  color: gray.300;
  vertical-align: middle;
}

.emotion-23 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  position: relative;
  width: full;
}

.emotion-24 {
  width: 100%;
  height: 150%;
  display: inline-block;
  line-height: 1em;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  color: red.50;
  vertical-align: middle;
  fill: none;
  position: absolute;
  top: -20%;
  left: 0;
  z-index: -1;
}

.emotion-25 {
  position: relative;
  height: 400px;
  border-radius: 2xl;
  box-shadow: 2xl;
  width: full;
  overflow: hidden;
}

.emotion-26 {
  object-fit: cover;
  object-position: center;
  width: 100%;
  height: 100%;
}

<div>
  <div
    class="chakra-container emotion-0"
  >
    <div
      class="chakra-stack emotion-1"
    >
      <div
        class="chakra-stack emotion-2"
      >
        <h2
          class="chakra-heading emotion-3"
        >
          <span
            class="chakra-text emotion-4"
          >
            LOGISTICO
          </span>
          <br />
        </h2>
        <h2
          class="chakra-heading emotion-5"
        >
          <span
            class="chakra-text emotion-6"
          >
            Your reliable logistics partner
          </span>
        </h2>
        <div
          class="emotion-7"
        >
          <div
            class="chakra-stack emotion-8"
          >
            <div
              class="chakra-stack emotion-8"
            >
              <label
                class="chakra-form__label emotion-10"
                for="downshift-0-input"
                id="downshift-0-label"
              >
                Choose drop-off locations
              </label>
              <div
                class="chakra-stack emotion-11"
              />
              <div
                aria-expanded="false"
                aria-haspopup="listbox"
                aria-owns="downshift-0-menu"
                class="chakra-stack emotion-12"
                role="combobox"
              >
                <input
                  aria-autocomplete="list"
                  aria-controls="downshift-0-menu"
                  aria-labelledby="downshift-0-label"
                  autocomplete="off"
                  class="chakra-input emotion-13"
                  id="downshift-0-input"
                  placeholder="Type an area"
                  value=""
                />
                <button
                  aria-label="toggle menu"
                  class="chakra-button emotion-14"
                  id="downshift-0-toggle-button"
                  tabindex="-1"
                  type="button"
                >
                  <svg
                    class="chakra-icon emotion-15"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      fill="currentColor"
                    />
                  </svg>
                </button>
              </div>
              <div
                class="emotion-16"
              >
                <ul
                  aria-labelledby="downshift-0-label"
                  class="emotion-17"
                  id="downshift-0-menu"
                  role="listbox"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          class="chakra-stack emotion-18"
        >
          <button
            class="chakra-button emotion-19"
            type="button"
          >
            Book delivery
          </button>
          <button
            class="chakra-button emotion-20"
            type="button"
          >
            <span
              class="chakra-button__icon emotion-21"
            >
              <svg
                aria-hidden="true"
                class="chakra-icon emotion-22"
                focusable="false"
                viewBox="0 0 58 58"
              >
                <path
                  d="M28.9999 0.562988C13.3196 0.562988 0.562378 13.3202 0.562378 29.0005C0.562378 44.6808 13.3196 57.438 28.9999 57.438C44.6801 57.438 57.4374 44.6808 57.4374 29.0005C57.4374 13.3202 44.6801 0.562988 28.9999 0.562988ZM39.2223 30.272L23.5749 39.7247C23.3506 39.8591 23.0946 39.9314 22.8332 39.9342C22.5717 39.9369 22.3142 39.8701 22.0871 39.7406C21.86 39.611 21.6715 39.4234 21.5408 39.1969C21.4102 38.9705 21.3421 38.7133 21.3436 38.4519V19.5491C21.3421 19.2877 21.4102 19.0305 21.5408 18.8041C21.6715 18.5776 21.86 18.3899 22.0871 18.2604C22.3142 18.1308 22.5717 18.064 22.8332 18.0668C23.0946 18.0696 23.3506 18.1419 23.5749 18.2763L39.2223 27.729C39.4404 27.8619 39.6207 28.0486 39.7458 28.2713C39.8709 28.494 39.9366 28.7451 39.9366 29.0005C39.9366 29.2559 39.8709 29.507 39.7458 29.7297C39.6207 29.9523 39.4404 30.1391 39.2223 30.272Z"
                  fill="currentColor"
                />
              </svg>
            </span>
            How It Works
          </button>
        </div>
      </div>
      <div
        class="emotion-23"
      >
        <svg
          class="chakra-icon emotion-24"
          focusable="false"
          viewBox="0 0 578 440"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M239.184 439.443c-55.13-5.419-110.241-21.365-151.074-58.767C42.307 338.722-7.478 282.729.938 221.217c8.433-61.644 78.896-91.048 126.871-130.712 34.337-28.388 70.198-51.348 112.004-66.78C282.34 8.024 325.382-3.369 370.518.904c54.019 5.115 112.774 10.886 150.881 49.482 39.916 40.427 49.421 100.753 53.385 157.402 4.13 59.015 11.255 128.44-30.444 170.44-41.383 41.683-111.6 19.106-169.213 30.663-46.68 9.364-88.56 35.21-135.943 30.551z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
        <div
          class="emotion-25"
        >
          <img
            alt="Hero Image"
            class="chakra-image__placeholder emotion-26"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
